packages/sdk/src/actions/actions.ts

```diff
@@ -69,12 +76,10 @@ const enhancePayloadForForwardToProxy = (payload: TSdkActionPayloadForUri) => {
       'X-Forward-To': payload.uri,
       'X-Forward-To-Audience-Policy':
         payload.audiencePolicy || 'forward-url-full-path',
-      ...(Boolean(payload.exchangeTokenClaims?.length)
+      ...(Boolean(exchangeTokenClaims?.length)
```

Comment:
Same here.

PS: we should use the `omitEmpty` function actually here as well. Could you add that? Thanks

```diff
-headers: {
+headers: omitEmpty<THeaders>({
```

Reply:
Updated in this commit: 7cd2e9fb90a205a7b229b646c5166a3c718b37bc